src/server/v2.0/handler/assembler/report.go

```diff
@@ -96,7 +96,8 @@ func (assembler *ScanReportAssembler) Assemble(ctx context.Context) error {
 			if err != nil {
 				log.Warningf("get scan summary of artifact %s@%s for %s failed, error:%v", artifact.RepositoryName, artifact.Digest, v1.MimeTypeSBOMReport, err)
 			}
-			if len(overview) == 0 {
+			// only fetch the sbom overview from execution when the overview is empty and the artifact has child references ( image index, cnab etc)
+			if len(overview) == 0 && len(artifact.References) > 0 {
```

Comment:
what if `len(overview) == 0` && `len(artifact.References) == 0`? In this case, line 128, overview is nil or empty map.